cypress/integration/sortTableDom.spec.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+describe('Sorting table app', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should sort in ASC order, by clicking on `Name` header', () => {
+    cy.contains('Name').click();
+    cy.get('tbody tr:nth-child(1) td:nth-child(1)').contains('Airi Satou');
+    cy.get('tbody tr:nth-child(2) td:nth-child(1)').contains('Colleen Hurst');
+    cy.get('tbody tr:nth-child(12) td:nth-child(1)').contains('Zorita Serrano');
```

Comment:
We are relying on the data. Let's read column data and make sure it's sorted?

Reply:
tests were significantly changed